utils/src/ooinstall/oo_config.py

```diff
@@ -176,6 +177,10 @@ def _read_config(self):
                 if 'Description' in self.settings:
```

Comment:
A bit unrelated to this PR, but would the legacy upgrade make more sense to key off the version field rather than 'Description'?